### DISCUSSION TITLE
Embed videos in Help Center articles?

### DISCUSSION THREAD

**Discussion by jordan-brough** [chatwoot/chatwoot#7435] [Feature Requests]
The new Help Center looks great. Is there a way to embed videos, or in general to add custom HTML to a help center page? I couldn't seem to figure out a way from the editor

**Comment by jxs714** [comment]
Any update on this? its important for us to add font colors to make text stand out, and to put URL links in the page

**Comment by jonaaix** [comment]
I would also appreciate to be able to embed any custom HTML snippet.

**Comment by sojan-official** [comment]
We support embedding youtube,vimeo, loom , any mp4 videos in the articles.  just paste the link into the article and it will be automatically converted to embeds on customer end.

**Reply by cognitus** [reply]
what we can do for insert content from other services? it would be usefull a snippet for insert custom html

**Comment by sojan-official** [comment]
Please raise additional requests as separate github issues !!!
